int8 CONTROLLER_STATUS_RUNNING=1
int8 CONTROLLER_STATUS_STOPPED=2
int8 CONTROLLER_STATUS_FAULTED=3
int8 CONTROLLER_STATUS_FIND_CONTACT=4
int8 CONTROLLER_STATUS_REFINE_CONTACT=5
int8 CONTROLLER_STATUS_FOUND_CONTACT=6
int8 CONTROLLER_STATUS_OPEN=7
int8 controller_status
